pulp_ansible/app/serializers.py

```diff
@@ -126,16 +127,21 @@ class AnsibleRepositorySerializer(RepositorySerializer):
     last_synced_metadata_time = serializers.DateTimeField(
         help_text=_("Last synced metadata time."), allow_null=True, required=False
     )
+    last_synced_time = serializers.SerializerMethodField()
     gpgkey = serializers.CharField(
         help_text="Gpg public key to verify collection signatures against",
         required=False,
         allow_null=True,
     )
 
+    def get_last_synced_time(self, obj):
+        return get_last_synced_task_time(obj.pk)
+
```

Comment:
I think we should fix the bug of `last_synced_metadata_time` always being `null` rather than adding a new field.

Reply:
I wasn't sure about this one. I thought it rather could be related to the `rebuil_metadata` action